metrics/metrics-integration/metrics-prometheus/src/main/java/org/apache/servicecomb/metrics/prometheus/MetricsPublisher.java

```diff
@@ -36,7 +36,7 @@
 
 @Component
 public class MetricsPublisher implements ApplicationListener<ApplicationEvent> {
-  private static final Logger LOGGER = LoggerFactory.getLogger(MetricsPublisher.class);
+  private static final Logger logger = LoggerFactory.getLogger(MetricsPublisher.class);
```

Comment:
why changed to logger? for static final varibles, use upper case, see this code below(METRICS_PROMETHEUS_PORT).

Reply:
DONE